packages/libinih/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "libinih"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "/dev/null"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/benhoyt/inih/archive/refs/tags/"
```

This link 404s for me.
```suggestion
releases-url = "https://github.com/benhoyt/inih/releases"
```

---

Good call out. I can fix this.